code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2021,6 +2033,61 @@
 	taste_description = "bananas"
 	can_synth = TRUE
 
+/datum/reagent/wittel
+	name = "Wittel"
+	description = "An extremely rare metallic-white substance only found on demon-class planets."
+	color = "#FFFFFF" // rgb: 255, 255, 255
+	taste_mult = 0 // oderless and tasteless
+
+/datum/reagent/metalgen
+	name = "Metalgen"
+	data = list("material"=null)
+	description = "A purple metal morphic liquid, said to impose it's metallic properties on whatever it touches."
+	color = "#b000aa"
+	taste_mult = 0 // oderless and tasteless
+	var/applied_material_flags = MATERIAL_ADD_PREFIX | MATERIAL_COLOR
+	var/minumum_material_amount = 100
+
+/datum/reagent/metalgen/reaction_obj(obj/O, volume)
+	metal_morph(O)
+	return
+
+/datum/reagent/metalgen/reaction_turf(turf/T, volume)
+	metal_morph(T)
+	return
+
+/datum/reagent/metalgen/proc/metal_morph(atom/A)
+	var/metal_dat = data["material"]
+	if(!metal_dat)
+		return
+	var/metal_amount = 0
+
+	for(var/B in A.custom_materials)
+		metal_amount += A.custom_materials[B]
+
+	if(!metal_amount)
+		metal_amount = minumum_material_amount
+
+	A.material_flags = applied_material_flags
+	A.set_custom_materials(list(getmaterialref(metal_dat) = metal_amount))
+
+/datum/reagent/gravitum
+	name = "Gravitum"
+	description = "A rare kind of null fluid, capable of temporalily removing all weight of whatever it touches." //i dont even
+	color = "#050096" // rgb: 5, 0, 150
+	taste_mult = 0 // oderless and tasteless
+	metabolization_rate = 0.1 * REAGENTS_METABOLISM //20 times as long, so it's actually viable to use
+	var/time_multiplier = 1 MINUTES //1 minute per unit of gravitum on objects. Seems overpowered, but the whole thing is very niche
+
+/datum/reagent/gravitum/reaction_obj(obj/O, volume)
+	QDEL_IN(new /datum/element/forced_gravity(O, 0), volume * time_multiplier)
```

Comment:
Did you test this at all, did you even bother looking up how other elements work? This is supreme shitcode, message me on discord or something if you don't know why

Reply:
Crap, yeah because Elements are a global thing and only one of them exists. I made this a few months ago before elements were a thing. I'll double check all the gravitum code